### DISCUSSION TITLE
Looking for a way to make t function typesafe

### DISCUSSION THREAD

**Discussion by yuloskov** [i18next/next-i18next#2065] [Q&A] [answered]
Hi! I'm looking for a way to make t function typesafe. Do you have an example for it? I'm looking for something like this https://react.i18next.com/latest/typescript

**Comment by adrai** [comment, marked as answer]
https://github.com/i18next/next-i18next/blob/master/examples/simple/%40types/i18next.d.ts

**Reply by yuloskov** [reply]
Thank you!
